Added information about pulling container images (#918)
RedHatInsights/red-hat-ansible-automation-platform-documentation: downstream/modules/platform/ref-network-ports-protocols.adoc

|link:https://automation-hub-prd.s3.amazonaws.com[https://automation-hub-prd.s3.amazonaws.com] |
|link:https://galaxy.ansible.com[https://galaxy.ansible.com] |Ansible Community curated Ansible content
|link:https://ansible-galaxy.s3.amazonaws.com[https://ansible-galaxy.s3.amazonaws.com] |
|link:https://registry.redhat.io:443[https://registry.redhat.io:443] |Access to container images provided by Red Hat and partners
|link:https://cert.cloud.redhat.com:443[https://cert.cloud.redhat.com:443] |Red Hat and partner curated Ansible Collections
|===

.Execution Environments (EE)
[options="header"]
|===
|URL |Required for
|link:https://registry.redhat.io:443[https://registry.redhat.io:443] |Access to container images provided by Red Hat and partners
| `cdn.quay.io:443` | Access to container images provided by Red Hat and partners
| `cdn01.quay.io:443` | Access to container images provided by Red Hat and partners
| `cdn02.quay.io:443` | Access to container images provided by Red Hat and partners
| `cdn03.quay.io:443` | Access to container images provided by Red Hat and partners
|===

[IMPORTANT]
====
Image manifests and filesystem blobs are served directly from `registry.redhat.io`. 

> However, from 1 of May 2023, filesystem blobs are served from `quay.io` instead. 

Reviewer: ```suggestion
However, from 1 May 2023, filesystem blobs are served from `quay.io` instead. 
```

> However, from 1st of May 2023, filesystem blobs are served from `quay.io` instead. 

Reviewer: ```suggestion
However, from 1 May 2023, filesystem blobs are served from `quay.io` instead. 
```

To avoid problems pulling container images, you must enable outbound connections to the listed `quay.io` hostnames.
Make this change to any firewall configuration that specifically enables outbound connections to `registry.redhat.io`.
Use the hostnames instead of IP addresses when configuring firewall rules. 
After making this change you can continue to pull images from `registry.redhat.io`. 
You do not need a `quay.io` login, or need to interact with the `quay.io` registry directly in any way to continue pulling Red Hat container images.

> For more information, see the article link:https://access.redhat.com/articles/6999582[here]

Reviewer: ```suggestion
For more information, see the article link:https://access.redhat.com/articles/6999582[here].
```
